pp/peers/start.go

```diff
@@ -29,8 +30,9 @@ func InitPeer(registerFn func()) {
 	//todo: register func call shouldn't be in peers package
 	registerFn()
 	GetSPList()
+	GetPPStatusFromSP()
 	//go SendLatencyCheckMessageToSPList()
-	InitPPList()
+	//InitPPList() // moved to rsp of GetPPStatusFromSP()
 	go ListenOffline()
 }
```

Comment:
Add a todo to understand the InitPeer function

Reply:
TODO added